CorsixTH/Lua/utility.lua

```diff
@@ -279,7 +279,7 @@ function compare_tables(t1, t2)
     if t2[k] ~= v then return false end
   end
   local count2 = 0
-  for k, v in pairs(t2) do
+  for _, _ in pairs(t2) do
```

Comment:
This looks weird, simpler would be  local count2 = #t2  and removal of the loop, as far as I can see